staging_service/import_specifications/individual_parsers.py

```diff
@@ -87,6 +115,15 @@ def _normalize(val: PRIMITIVE_TYPE) -> PRIMITIVE_TYPE:
     return val
 
 
+def _normalize(val: str) -> PRIMITIVE_TYPE:
+    val = val.strip()
+    try:
+        num = float(val)
+        return int(num) if num.is_integer() else num
+    except ValueError:
+        return val if val else None
```

Comment:
Is there an example of a ValueError in which a "val" would be returned instead of None?

Reply:
If `val` is any non-empty string it'll be returned

Reply:
Can we add a comment describing the purpose of this normalization?

Reply:
done